renpy/display/layout.py

```diff
@@ -2228,55 +2228,50 @@ class AlphaMask(Container):
     The `child` and `mask` parameters may be arbitrary displayables. The
     size of the AlphaMask is the size of `child`.
 
+    `alpha_over` is a float that ranges from 0.0 to 1.0.
+    At `0.0`, nothing changes. At `1.0`, the behavior is swapped (`child` is opaque where `mask` is transparent).
```

Comment:
"Nothing changes" is inaccurate : with 0.0, the alphamask still does its job and the child gets masked. Passing 0.0 to [InvertMatrix](https://www.renpy.org/dev-doc/html/matrixcolor.html#InvertMatrix) is a situation where we can actually say nothing changes.

Also, I think it should be introduced and documented as a boolean defaulting to False and which inverts the mask, and then you maybe say you can pass a float to tweak things more specifically (but it seems like a weird idea to pass anything other than 1 or 0 so it could stay undocumented as far as I'm concerned).

Reply:
i meant that the behavior stays the same

Reply:
> And I'm also not sure about the name. invert_mask ?
i was thinking of `invert_alpha` but i guess the meaning is the same

Reply:
> but it seems like a weird idea to pass anything other than 1 or 0

one such use case would be some sort of animated displayable where the value is interpolated. if you're just passing a static value, then yes, not a very bright idea since only `0.0` and `1.0` have a good looking effect